organization/components/timeslot/TimeslotForm.tsx

```
@@ -0,0 +1,163 @@
+            min={formData.startDate || new Date().toISOString().split('T')[0]}
+            required
+          />
+        </div>
+      </div>
+
+      {/* Slot Duration */}
+      <div className="space-y-1">
+        <label className="text-sm font-medium text-gray-700 flex items-center gap-2">
+          <FiClock className="text-blue-600" />
+          Slot Duration (minutes)
+        </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)